api/app/controllers/api/v1/ingests_controller.rb

```diff
@@ -0,0 +1,53 @@
+module API
+  module V1
+    # Trigger ingestion processing via the API
+    class IngestsController < ApplicationController
```

Comment:
Because this creates an `Ingestion` model, the controller should be called `IngestionsController` to be proper and restful